bigtop-tests/spec-tests/runtime/src/test/resources/testRuntimeSpecConf.groovy

```diff
@@ -218,7 +218,7 @@ specs {
     }
     'HADOOP_BINCONTENT_COMMON' {
       name = 'HADOOP_BINCONTENT_COMMON'
-      type = 'dircontent'
+      type = 'dirstruct'
```

Comment:
It basically changes the type check from having an exact directory match, to only ensuring the tested system has the same files as the reference does ( allowing extra files to be in there ).

You'll want to check https://github.com/odpi/bigtop/blob/odpi-master/bigtop-tests/spec-tests/runtime/src/test/groovy/org/odpi/specs/runtime/TestSpecsRuntime.groovy to see more details.